lib/core/src/lib/comments/comments.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="adf-comments-container">
-    <h1 id="comment-header" class="adf-comments-header">
+    <h2 id="comment-header" class="adf-comments-header">
```

Comment:
I'm still not sure what this change is fixing? according to the semantics, h2 should be declared after h1, but h1 is missing here